source/extensions/filters/network/dubbo_proxy/router/route_matcher.cc

```diff
@@ -0,0 +1,276 @@
+#include "extensions/filters/network/dubbo_proxy/router/route_matcher.h"
+
+#include "envoy/config/filter/network/dubbo_proxy/v2alpha1/dubbo_proxy.pb.h"
+
+#include "common/protobuf/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+namespace Router {
+
+bool Utility::isContainWildcard(const std::string& input) {
+  return (input.find('*') != std::string::npos) || (input.find('?') != std::string::npos);
```

Comment:
https://en.wikipedia.org/wiki/Glob_(programming)  
Do you want to support the full wildcard?  or you just want to support the `*` and `?` ?, I suggest that it is the same as the virtual host, only `*` is supported.

Reply:
Dubbo routing matches need to support wildcard * and ?.